package.json

```diff
@@ -65,7 +65,10 @@
     "transform": {
       "^.+\\.json5?$": "json5-jest",
       "^.+\\.jsx?$": "babel-jest"
-    }
+    },
+    "transformIgnorePatterns": [
+      "^.+\\.json?$"
```

Comment:
Did you mean `"^.+\\.json$"` without the question mark?

The question mark makes the `n` optional.

Reply:
Good catch! I made the change below and removed transformIgnorePatterns completely and it works! 👍